homeassistant/components/zwave/workaround.py

```diff
@@ -68,28 +68,19 @@
 
 def get_device_component_mapping(value):
     """Get mapping of value to another component."""
-    if (value.node.manufacturer_id.strip() and
-            value.node.product_type.strip()):
```

Comment:
I don't think this was ever actually doing anything. `strip()` returns a stripped copy of the string. Since there's always a string object returned it always evaluates to true. It doesn't modify the original string.